integration/test/chartvalues/chartvalues_test.go

```diff
@@ -0,0 +1,44 @@
+// +build k8srequired
+
+package chartvalues
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/giantswarm/micrologger"
+
+	"github.com/giantswarm/chart-operator/integration/chart"
+	"github.com/giantswarm/chart-operator/integration/templates"
+)
+
+func TestChartValues(t *testing.T) {
+	const cr = "chart-operator-resource"
+
+	charts := []chart.Chart{
+		{
+			Channel: "1-0-beta",
+			Release: "1.0.0",
+			Tarball: "/e2e/fixtures/tb-chart-1.0.0.tgz",
+			Name:    "tb-chart",
+		},
+	}
+
+	// Setup
+	l, err := micrologger.New(micrologger.Config{})
+	if err != nil {
+		t.Fatalf("could not create logger %v", err)
+	}
+
+	err = chart.Push(f, charts)
+	if err != nil {
+		t.Fatalf("could not push inital charts to cnr %v", err)
+	}
+
+	// Test Creation
+	l.Log("level", "debug", "message", fmt.Sprintf("creating %s", cr))
+	err = f.InstallResource(cr, templates.ChartOperatorResourceValues, ":stable")
```

Comment:
A resource needs to be installed because of the current implementation of `teardown`. This will change in the future but this PR was already getting quite big.